src/mame/drivers/mc10.cpp

```diff
@@ -237,8 +228,27 @@ TIMER_DEVICE_CALLBACK_MEMBER(mc10_state::alice32_scanline)
     DRIVER INIT
 ***************************************************************************/
 
-DRIVER_INIT_MEMBER(mc10_state,mc10)
+mc10_state::mc10_state(const machine_config &mconfig, device_type type, const char *tag)
+	: driver_device(mconfig, type, tag)
+	, m_maincpu(*this, "maincpu")
+	, m_mc6847(*this, "mc6847")
+	, m_ef9345(*this, "ef9345")
+	, m_dac(*this, "dac")
+	, m_ram(*this, RAM_TAG)
+	, m_cassette(*this, "cassette")
+	, m_printer(*this, "printer")
+	, m_pb(*this, "pb%d", 0)
```

Comment:
If you look at this template, it uses `unsigned` everywhere.  Using `%d` with that is going to do weird casting, and you probably shouldn't be trying to stuff signed numbers through it unless you're using `%+d` to force the sign anyway.